packages/daemon/src/pet-store.js

```diff
@@ -104,22 +105,59 @@ export const makePetStoreMaker = (filePowers, locator) => {
       const petNamePath = filePowers.joinPath(petNameDirectoryPath, petName);
       const petNameText = `${formulaIdentifier}\n`;
       await filePowers.writeFileText(petNamePath, petNameText);
-      changesTopic.publisher.next({ add: petName });
+      const formularIdentifierPair = parseFormulaIdentifier(formulaIdentifier);
+      changesTopic.publisher.next({
+        add: petName,
+        value: formularIdentifierPair,
```

Comment:
Let’s call it a `FormulaIdentifierRecord` or `FormulaIdentifierEntry`. Pair is fragile if we add a `subtype` field or whatnot.